Tools/scripts/build_options.py

```diff
@@ -106,4 +106,14 @@ def __init__(self, category, label, define, description, default, dependency):
     Feature('Other', 'AIRSPEED', 'AP_AIRSPEED_ENABLED', 'Enable Airspeed Sensors', 1, None),    # Default to enabled to not annoy Plane users
     ]
 
+    Feature('GPS Drivers', 'SBP2', 'AP_GPS_SBP2_ENABLED', 'Enable SBP2 GPS', 0, 'SBP'),
+    Feature('GPS Drivers', 'SBP', 'AP_GPS_SBP_ENABLED', 'Enable SBP GPS', 0, None),
+    Feature('GPS Drivers', 'ERB', 'AP_GPS_ERB_ENABLED', 'Enable ERB GPS', 0, None),
+    Feature('GPS Drivers', 'GSOF', 'AP_GPS_GSOF_ENABLED', 'Enable GSOF GPS', 0, None),
+    Feature('GPS Drivers', 'NEMA', 'AP_GPS_NEMA_ENABLED', 'Enable NEMA GPS', 0, None),
```

Comment:
```suggestion
    Feature('GPS Drivers', 'NMEA', 'AP_GPS_NMEA_ENABLED', 'Enable NMEA GPS', 0, None),
```

Reply:
my dsylexia strikes again....wish we had some kind of CI for this...will fix